tools/msvs/msi/product.wxs

```diff
@@ -19,10 +19,14 @@
            UpgradeCode="47c07a3a-42ef-4213-a85d-8f5a59077c28">
 
     <Package Languages="!(loc.LocaleId)"
-             InstallerVersion="200"
+             InstallerVersion="500"
              Compressed="yes"
              InstallScope="perMachine"/>
 
+    <Condition Message="This application is only supported on Windows 7, Windows Server 2008 R2, or higher.">
+      <![CDATA[Installed OR (VersionNT >= 601)]]>
```

Comment:
This works as a negative condition? if this is false then the message is displayed?

Also, does `Installed` here mean that if you already have Node installed and try to install v6 then it's not going to give you this error?